ProjectCoimbra.UWP/Project.Coimbra/Pages/CreateAccountPage.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<Page
+    x:Class="Coimbra.Pages.CreateAccountPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource SystemControlTransientBackgroundBrush}">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition />
+            <RowDefinition />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition />
+            <ColumnDefinition />
+            <ColumnDefinition />
+        </Grid.ColumnDefinitions>
+        <StackPanel Grid.Column="1" Grid.RowSpan="3" Background="white"></StackPanel>
+        <StackPanel Grid.Column="1" Grid.Row="0">
+            <TextBlock x:Uid="LoginPage.SignIntoAccount" Text="COIMBRA" 
+                       Foreground="LightBlue" Style="{StaticResource HeaderTextBlockStyle}" HorizontalAlignment="Center" Padding="0,65,0,0" FontSize="70"/>
+            <TextBlock x:Uid="LoginPage.SignIntoAccount" Text="Create Account!" 
```

Comment:
I don't think it's being applied everywhere, but if we can render the text of the buttons using the resource values it would be better for localization efforts later on

Reply:
That is happening through the x:Uid tag, we just need to make sure every XAML tag with strings has one and that there is a corresponding entry in the resource file with the attribute that should be replaced.